Sming/Core/Data/Stream/JsonObjectStream.h

```diff
@@ -51,12 +51,11 @@ class JsonObjectStream : public MemoryDataStream
 	 */
 	int available() override
 	{
-		return rootNode.success() ? rootNode.measureLength() : 0;
+		return (doc.isNull() ? 0 : measureJson(doc));
```

Comment:
I was wondering whether the null test was necessary, so I tested `measureJson()` on an empty document - returns 4! ("null"). Which leads to another question of what constitutes a valid 'empty' JSON document. Technically, it should probably be `{}` though there is some disagreement on that... I wonder what a browser would accept?